### ISSUE TITLE
Content below local asciinema is hidden

### ISSUE THREAD

**Issue by zerocewl** [divi255/sphinxcontrib.asciinema#15]
NOTE: This bug comes with changes from 0.2.1 to 0.3.1.

After an update to version 0.3.1 the content below an `asciinema` element is hidden.
It is using a local file.

If the generated html `<asciinema-player ...` is manually wrapped into a container like `<p>`, `<span>` or `<div>` the content is visible again.

Maybe the `<asciinema-player {options} src="data:application/json;base64,{src}" />` should be moved into a container?

Available workarounds:
* downgrade to 0.2.1
* move the ``asciinema`` into a container by hand e.g.

```rst
.. container:: custom

   .. asciinema:: examples/test.cast
      :title: test me

I'm hidden if the container is removed.
```

**Comment by divi255** [comment]
Hi. Sorry for the delay, will take a look soon

**Comment by divi255** [comment]
it looks <asciinema-player> tag just needs to be closed in HTML way. fixed in 0.3.2
